salt/modules/cassandra_mod.py

```diff
@@ -27,6 +34,13 @@
     pass
 
 
+warn_until_date(
```

Comment:
Won't this be issued everytime the loader tried to load it?
That might be a tad too noisy.

How about a decorator that triggers the warning if any of the functions are used?